### DISCUSSION TITLE
1. Can two or more wired pads be connected at the same time?

### DISCUSSION THREAD

**Discussion by fayehoo** [cathery/sys-con#227] [Q&A] [answered]
1. Can two or more wired pads be connected at the same time?

2. I've tested the powerA pad and it works. But Gamesir pad not work. PC compatible pad（non Xbox type）does not be supported yet?  Is it possible to support other 3rd-party pad  in the future?

**Comment by cathery** [comment, marked as answer]
1. yes
2. only supported controllers are supported
